orangecontrib/geo/geojson/admin1-SVN.json

```diff
@@ -5,8 +5,8 @@
-{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[15.335064324758747,46.47739594216898],[15.338474969444292,46.465432848247275],[15.352479281593588,46.45256541586119],[15.398367954317592,46.437889309244184],[15.422759230154384,46.42460846570816],[15.409891798667559,46.40548818653053],[15.419038527106409,46.39329254771286],[15.409116652311667,46.3808902042195],[15.425239698853005,46.371252550264956],[15.412217237735376,46.354845282883446],[15.388136021160506,46.35629222280784],[15.368757357765816,46.33396800451976],[15.338785027806722,46.34936758154859],[15.331188592079883,46.365206407249815],[15.354029575204748,46.38308645297738],[15.358370395877103,46.39763336928456],[15.325297478695745,46.429207668798796],[15.326847772306905,46.43907786675018],[15.336924675833188,46.44589915612116],[15.324832390702284,46.45124766714616],[15.327002801038446,46.45853404451054],[15.317701042968793,46.46972199207627],[15.31568566316281,46.482176012412964],[15.335064324758747,46.47739594216898]]]},"properties":{"latitude":46.3914,"_id":"SVN-1007","iso_a2":"SI","adm0_a3":"SVN","type":"Commune|Municipality","hasc":"SI.SA.ZR","longitude":15.3705,"admin":"Slovenia","name":"Zrece","fips":"SIF3"}},
+{"type":"Feature","geometry":{"type":"Polygon","coordinates":[[[14.970745477221953,46.19777476739159],[14.999890984881006,46.198989163319226],[15.012810093211243,46.182194322310096],[14.985938347776568,46.166794745281265],[14.999115837625823,46.16806081805231],[15.013430209936217,46.155916856077994],[15.027072787778934,46.14018138316433],[15.03125857882037,46.125918687697265],[15.01467044428557,46.11666860647094],[14.999890984881006,46.11909739832623],[14.976171501713452,46.11219859548879],[14.951470168413579,46.097393296763215],[14.911317580342086,46.09287160893754],[14.892352328996793,46.10886546516903],[14.890491977922466,46.12777903877162],[14.839332310337852,46.12958771390191],[14.834784784090544,46.1369774440538],[14.840572543787829,46.14896637639711],[14.828428581813597,46.16028351517204],[14.816801384676154,46.17051544922859],[14.892662388258628,46.179248766517304],[14.901705763010511,46.19097931644225],[14.898915235949346,46.20867849411718],[14.910077345093441,46.210538845191536],[14.915348341752605,46.201366279230456],[14.937362501678052,46.19185781468636],[14.970745477221953,46.19777476739159]]]},"properties":{"latitude":46.1454,"_id":"SVN-1006","iso_a2":"SI","adm0_a3":"SVN","type":"Statistical Region","hasc":"SI.ZS","longitude":14.924,"admin":"Slovenia","name":"Zagorje ob Savi","fips":"SIE7"}},
```

Comment:
This line is about the [statistical region](https://en.wikipedia.org/wiki/Statistical_regions_of_Slovenia) _Zasavska_ (!= municipality _Zagorje ob Savi_).